logo/files

```diff
@@ -0,0 +1 @@
+
```

Comment:
What is this file "files" about?

Reply:
I had to create a folder to upload the logo files, but I forget to delete it, sorry again :)

Reply:
No worries :)